CMakeLists.txt

```diff
@@ -696,6 +696,12 @@ if (BUILD_TESTING)
             set(ARTIFACT_UPLOAD_LOC "none")
         endif()
 
+        if(COVERAGE)
+            set(FUZZ_COVERAGE "true")
```

Comment:
Is there a reason that we can't just use `COVERAGE`?

Reply:
You're right, I can just use `COVERAGE` :sweat:. Thank you for pointing it out